### ISSUE TITLE
mask_logits in layer.py

### ISSUE THREAD

**Issue by zhaoluffy** [minsangkim142/Fast-Reading-Comprehension#43]
I think the line "return inputs + mask_value * (1 - mask)"  should be "return inputs*mask + mask_value * (1 - mask)"

**Comment by localminimum** [comment]
You are correct, however, I believe that the difference is very minuscule (probably even smaller than a floating point epsilon) so I saw no point in fixing it.
